TCMSApp/src/client/app/defects/defects.controller.js

```diff
@@ -5,15 +5,29 @@
         .module('app.defects')
         .controller('DefectsController', DefectsController);
 
-    DefectsController.$inject = ['$scope', 'logger', '$uibModal'];
+    DefectsController.$inject = ['$scope', 'logger', '$uibModal', 'getDefects'];
     /* @ngInject */
-    function DefectsController($scope, logger, $uibModal) {
+    function DefectsController($scope, logger, $uibModal, getDefects) {
         var vm = this;
 
+        $scope.arrayDefects = getDefects.fakeDefect(10);
+        var store = $scope.arrayDefects[0];
+        $scope.getDefects = function (x) {
```

Comment:
It is not so good to make the name of variable the same as name of service. It may cause problems